docs/source/tutorials/open_images_evaluation.ipynb

```diff
@@ -0,0 +1,444 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Object Detection Model Evaluation on Google's Open Images Dataset\n",
+    "\n",
+    "This tutorial demonstrates per-image evaluation of an object detection model on [the Open Images dataset](https://storage.googleapis.com/openimages/web/index.html)\n",
+    "that generates:\n",
+    "\n",
+    "- true positives & false positives\n",
+    "- per-class average precision (AP)\n",
+    "- mean average precision (mAP)\n",
+    "\n",
+    "for each image and adds this information to each [Sample](https://voxel51.com/docs/fiftyone/api/fiftyone.core.sample.html#fiftyone.core.sample.Sample)\n",
+    "in the [Dataset](https://voxel51.com/docs/fiftyone/api/fiftyone.core.dataset.html#fiftyone.core.dataset.Dataset).\n",
+    "\n",
+    "The steps are broken down as follows:\n",
+    "\n",
+    "1. [Requirements](#Requirements)\n",
+    "2. [Download the test data and ground-truth labels](#Download-the-test-data-and-ground-truth-labels) (optional)\n",
+    "3. [Generate predictions](#Generate-predictions) (optional)\n",
+    "4. [Load the data into FiftyOne](#Load-the-data-into-FiftyOne)\n",
+    "5. [Prepare the ground-truth for evaluation](#Prepare-the-ground-truth-for-evaluation) (optional)\n",
+    "6. [Evaluate on a per-image granularity](#Evaluate-on-a-per-image-granularity)\n",
+    "7. [Explore](#Explore)\n",
+    "\n",
+    "Optional steps may not be necessary depending on if you have already downloaded the data or have your own model to evaluate.\n",
+    "\n",
+    "This tutorial evaluates a model on [Open Images V4](https://storage.googleapis.com/openimages/web/download_v4.html)\n",
+    "however this code supports later versions of Open Images as well. If using a newer version just make sure to\n",
+    "use the appropriate hierarchy file and class label map."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Requirements\n",
+    "\n",
+    "This notebook contains bash commands. To run it as a notebook, you must install the [Jupyter bash kernel](https://github.com/takluyver/bash_kernel) via the command below.\n",
+    "\n",
+    "Alternatively, you can just copy + paste the code blocks into your shell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pip install bash_kernel\n",
```

Comment:
I wasn't able to get this to play nice with my existing virtual environment so I had to reinstall a bunch of packages. I'd recommend just having users copy and paste the commands into their console. If you do wanna go with this then probably add instructions of how to use the `Bash` kernel that is created (e.g. "click on the Kernels option in jupyter",...)

Also when I started this notebook, it said that it could not find a kernel called `Bash` because I hadn't installed it yet. Not sure where that is saved in the notebook but the default `Bash` kernel should probably be removed.

Reply:
I updated this description. Let me know if it's more clear

Reply:
The description is good. But see my other comments about maybe removing the `Bash` kernel altogether.